Assets/Scripts/ScriptableObjects/EnemiesSystem/EnemyAttackStrategies/KamikazeStrategy.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Player;
+using UnityEngine;
+
+namespace ScriptableObjects.EnemiesSystem.EnemyAttackStrategies
+{
+    [CreateAssetMenu(fileName = "KamikazeAttackStrategy", 
+        menuName = "ScriptableObjects/EnemyAttack/kamikaze", order = 3)]
+    public class KamikazeStrategy : EnemyAttackStrategy
+    {
+
+        [SerializeField] private GameObject bloodEffect;
+        
+        public override bool Attack(GameObject player, GameObject emu, int damageGiven, bool hasCollided)
+        {
+            if (player.CompareTag("Player") && hasCollided)
+            {
+                var playerController = player.GetComponent<Controller>();
+                playerController.LoseHitPoints(damageGiven);
+                Instantiate(bloodEffect, emu.transform.position, emu.transform.rotation);
+                Destroy(emu);
```

Comment:
Idk why but this line is hilarious to me

Reply:
hehehe